unstructured_ingest/v2/processes/connectors/neo4j.py

```diff
@@ -110,6 +110,7 @@ def run(  # type: ignore
         return output_filepath
 
     def _create_lexical_graph(self, elements: list[dict], document_node: _Node) -> nx.Graph:
+        import networkx as nx
```

Comment:
Does this need to be added to the neo4j requirements file then?

Reply:
I agree, making a pip compile now